apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -91,13 +91,15 @@ export const styles = {
 const SectionsTable = React.createClass({
   propTypes: {
     sections: shapes.sections,
-    isRtl: React.PropTypes.bool.isRequired,
+    isRtl: PropTypes.bool.isRequired,
     isTeacher: PropTypes.bool.isRequired,
-    canLeave: PropTypes.bool.isRequired
+    canLeave: PropTypes.bool.isRequired,
+    updateSections: PropTypes.func.isRequired
   },
 
-  onLeave() {
-    //TODO: Clicking this button should remove the student from that section. Remove display: 'none' from leaveCol styles so the button will appear for students who have permission to leave sections.
+  onLeave(sectionCode) {
+    $.post(`/api/v1/sections/${sectionCode}/leave`)
+      .done(data => this.props.updateSections(data.sections));
```

Comment:
jquery does message the 404 to the console but this still happens even if I add a `.fail` handler.